### ISSUE TITLE
CLI: Porter config management commands

### ISSUE THREAD

**Issue by vdice** [getporter/porter#1077]
**Is your feature request related to a problem? Please describe.**
Currently, [Porter's config file](https://porter.sh/configuration/) can be created/edited manually via its expected location (`~/.porter/config.toml`).

It would be nice if we had a `porter config` suite of CLI commands to create, edit and view this file, without necessarily directly accessing its location mentioned above.

**Describe the solution you'd like**
A new suite of commands under `porter config`:
 - `porter config create` - to create the configuration file (not sure if Porter creates it behind-the-scenes already?)  I might be convinced that this subcommand isn't necessary (Perhaps `porter config edit` would just create the file if it didn't already exist.)
 - `porter config show` - to show the current configuration
 - `porter config edit` - to edit the current configuration
 - Other? Different?

**Describe alternatives you've considered**
As mentioned in the description, the alternative is to create/open/edit the file via its location in the Porter home directory.

**Additional context**
N/A

**Comment by carolynvs** [comment]
That's right, porter doesn't create the config file for you, it's a manual step.

I'm going to put this issue on the backburner for now. As you noted it's not ready to work on yet, and is much lower priority than our v1 milestone. We can come back to this later after the v1 release.

**Comment by vdice** [comment]
> @vdice CMIIW but I don't think that Porter created it automatically behind-the-scenes (?) I checked my ~/.porter directory and couldn't find any config file, in .toml format or other formats possible mentioned on the docs.

Sorry @joshuabezaleel , missed this comment!  You are correct, thanks for the correction.
